src/test/resources/testBadTechnique.xml

```diff
@@ -0,0 +1,28 @@
+<TECHNIQUE name="Test technique">
+  <DESCRIPTION>A test technique</DESCRIPTION>
+
+  <SYSTEM>true</SYSTEM>
+
+  <MULTIINSTANCE>false</MULTIINSTANCE>
+
+  <TMLS>
+  </TMLS>
+
+  <SYSTEMVARS>
+  </SYSTEMVARS>
+
+  <TRACKINGVARIABLE>
+  </TRACKINGVARIABLE>
+
+  <SECTIONS>
+    <SECTION name="common">
+      <INPUT>
+        <NAME>A</NAME>
+        <DESCRIPTION>...</DESCRIPTION>
+        <CONSTRAINT>
+          <USES>missing,VARIABLES</USES>
```

Comment:
yes, so much not use that it is not parsed at all. And the test was removed, because it was unfasifiable given the new production mecanism :)